appservice/src/SiteClient.ts

```diff
@@ -84,89 +82,89 @@ export class SiteClient {
     }
 
     public async stop(): Promise<void> {
-        this.isSlot ?
+        this.slotName ?
```

Comment:
So why the change to checking slotName instead of isSlot?  Don't like that there are two members dependent on each other but there is no assertion verifying this dependence.

Reply:
I can change `isSlot` to be `!!this.slotName`. There's no need to have different logic.

The change was so that the compiler wouldn't complain when passing `this.slotName` to `this._client.webApps.startSlot`, which doesn't allow `undefined`